c2corg_ui/static/js/documentediting.js

```diff
@@ -151,6 +151,12 @@ app.DocumentEditingController = function($scope, $element, $attrs, appLang,
   this.submit = false;
 
   /**
+   * @type {Date}
+   * @export
+   */
+  this.today = new Date();
```

Comment:
> Maybe we could use a mako variable and render it with pyton today = ${today} ? 

That's an option.

> why is it more appropriate to set it in the template rather than here?

Because it is needed only for images, not the other doc types.

I think you should create an ``ImageEditing`` controller. You could move this property here as well.